### PR TITLE
Create account_takeover_application

### PR DESCRIPTION
**Pull request by lizygeogy**

### CODE REVIEW

**Comment by lizygeogy** [comment]
@DT-Priyanka 
Please verify and merge content.

File: docs/Accounttakeover.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+Account Takeover (ATO) Prevention application is a pre-built application that can be used to detect and prevent attempts of account takeover in various industries such as finance, telecom, and subscription-based offerings. ATO application is designed to ingest, transform, analyze incoming data, and provide account takeover alerts in real-time before such an activity occurs. It also provides visualizations of incoming as well as processed data.
+
+Using this application, you can process, enrich, analyze and act in real-time upon multiple streams of account event information which can prevent account-take over and fraud.
+
+ATO Prevention application is built over DataTorrent RTS platform and can be run on commodity hardware. The platform provides real time insights and a fault tolerant and scalable method for processing data. The application can be further customized by writing rules as per your business needs and implementing custom data types.
+
+In addition, ATO has the capability to store and replay the incoming data from Kafka input operators and then replay the stored data with a different set of rules to visualize the outcome. Refer _Store and Replay
```
Comment (wvary) [inline docs/Accounttakeover.md]: @lizygeogy This seems incomplete "Refer _Store and Replay".

**Comment by lizygeogy** [comment]
@DT-Priyanka 
Added images. Please verify and merge.

**Comment by sanjaypujare** [comment]
@DT-Priyanka  if you are happy can you merge?

File: docs/Accounttakeover.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+
+| **Property** | **Description** | **Type** | **Example** | **Required** |
+| --- | --- | --- | --- | --- |
+| kafkaBrokerList | Comma separated list of kafka-brokers | String | node1.company.com:9098, node2.company.com:9098, node3.company.com:9098 | Yes |
+| UserActivityReceiverTopic | Topics to read from Kafka | String | transactions | Yes |
+| ProcessedTransactionsOutputTopic | Topics to write processed transactions to kafka | String | processed-transactions | Yes |
+| initialOffset | Initial offset to read from Kafka | String | EARLIESTLATESTAPPLICATION\_OR\_EARLIESTAPPLICATION\_OR\_LATEST |   |
+| key.serializer | Serializer class | String | org.apache.kafka.common.serialization.StringSerializer |   |
+| value.serializer | Serializer class | String | org.apache.kafka.common.serialization.StringSerializer |   |
+| dt.operator.TransactionDataPublisher.prop.properties(key.serializer) | Serializer class | String | org.apache.kafka.common.serialization.StringSerializer |   |
```
Comment (DT-Priyanka) [inline docs/Accounttakeover.md]: In next phase we should standardise on how do we put properties in document. Let's keep it this way for now.

File: docs/Accounttakeover.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+3. Launch the application. 
+During the launch process, you can name the configuration and save it for future references. After you launch, you can track the details of the processed data in the application from the **Monitor** tab.
+
+## Generate Sample Input
+
+For a test run, you may want to generate sample data.
+
+To generate sample data, do the following:
+
+1. Run the **dt-ato-datagen-1.4.0.apa** application from the DataTorrent RTS interface.
```
Comment (DT-Priyanka) [inline docs/Accounttakeover.md]: This is now changed. The datagen is part of "dt-cep-omni-fraud-prevention-app-1.4.0.apa" So you can say launch "UserActivityGenerator" app from "dt-cep-omni-fraud-prevention-app-1.4.0.apa" package.

**Comment by lizygeogy** [comment]
@DT-Priyanka added the comments, please verify and close.

**Comment by DT-Priyanka** [comment]
@lizygeogy  can you please resolve conflict?

**Pull request merged**